packages/vue-vuetify/src/util/composition.ts

```diff
@@ -141,7 +142,7 @@ export const useVuetifyControl = <
   },
   I extends {
     control: ComputedRef<T>;
-  } & DispatchPropsOfControl,
+  } & (DispatchPropsOfControl | DispatchPropsOfMultiEnumControl),
```

Comment:
Seems weird to add this type here. What is the rationale?

Reply:
Hmm I see. The core issue is that we do not expose `handleChange` to these controls although I would have expected it. See for example the React renderers in which we not only do the specialized `addIterm` etc. but also expose the usual `handleChange`. 

I'm fine with this for this PR but we should rework in the future I think